Add Spring Boot registration in Java getting started

twsouthwick opened the pull request:
This adds a section on how to register the HTTP filter for Java Spring Boot applications, since they have their own MVC configuration

azurecla commented:
Hi __@arunvinoth__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

twsouthwick committed:
Add Spring Boot registration in Java getting started

This adds a section on how to register the HTTP filter for Java Spring Boot applications, since they have their own MVC configuration

dhaval24 reviewed (commented):
@twsouthwick this changes are only the part of things that need to properly configure AI with Spring boot. There are still known issues for out of box configuration of AI with spring boot and therefore I think it would be wise to have a blog post instead of putting it on official docs.
